config/middleware.go

```diff
@@ -48,6 +49,7 @@ func init() {
 	register("ext", extensions.New)
 	register("basicauth", basicauth.New)
 	register("proxy", proxy.New)
+	register("git", git.New)
```

Comment:
Good. I think this is a good place in the chain for this middleware to be registered.